controllers/grid/articleGalleys/ArticleGalleyGridCellProvider.php

```diff
@@ -64,6 +65,10 @@ public function getTemplateVarsFromRowColumn($row, $column)
                 return [
                     'label' => !$element->getData('urlRemote') && $element->getData('submissionFileId') ? '' : $element->getLabel()
                 ];
+            case 'language':
+                return [
+                    'label' => Locale::getSubmissionLocaleDisplayNames([$element->getLocale()])[$this->getLocale()]
```

Comment:
I think this should be:
`Locale::getSubmissionLocaleDisplayNames([$element->getLocale()], $this->getLocale())[$element->getLocale()]`

The array returned from the function has submission languages as keys, not the UI languages.
E.g. if the UI language is fr_CA, then there will be a warning: Undefined array key "fr_CA".

Reply:
ah true, thanks